crates/compiler/can/src/desugar.rs

```diff
@@ -143,6 +143,218 @@ pub fn desugar_defs<'a>(
     }
 }
 
+fn desugar_defs_node_suffixed<'a>(
+    arena: &'a Bump,
+    loc_expr: &'a Loc<Expr<'a>>,
+) -> &'a Loc<Expr<'a>> {
+    match loc_expr.value {
+        Defs(defs, loc_ret) => {
+            match defs.search_suffixed_defs() {
+                None => loc_expr,
+                Some((tag_index, value_index)) => {
+                    if defs.value_defs.len() == 1 {
+                        // We have only one value_def and it must be Suffixed
+                        // replace Defs with an Apply(Task.await) and Closure of loc_return
+
+                        assert!(
```

Comment:
I'd use `debug_assert` here - it works like `expect` in Roc in that it only runs in debug builds, not in release builds, which I think is what we want here!

Reply:
Oh nice, that was what I thought assert did. Thanks